### PR TITLE
Clean up SLURM, add deploy check, prep for full deployment

### LINKED ISSUE #55: Change prod flags to use single DEV flag
We have a couple flags for production, instead, we can use a single flag, either the existing DEBUG or a new DEV flag to reduce the complexity of the .env file. 

NOTE: Change would require updating of the admin manual
https://github.com/birdspotter-project/COS397/blob/c829d1e84da7f0cdc39bbba3d7624cc65eb5ad23/docker-compose/.env.example#L8

### PR DESCRIPTION
**Pull request by devinchristianson**

### CODE REVIEW

**Comment by devinchristianson** [comment]
Note: this PR isn't actually ready to merge, as it turns out I screwed up what the defaults were (so it's defaulting to a Postgres database in the CI)

**Comment by NickKania** [comment]
Update documentation
- [ ] Update docs to reflect flag change
- [ ] Update .env example

File: birdspotter/birdspotter/settings.py [hunk 1]
```diff
@@ -33,17 +33,27 @@
...
 TEST_RUNNER = 'birdspotter.TestRunner.TestRunner'
 
 ALLOWED_HOSTS = []
 USE_X_FORWARDED_HOST = os.getenv('USE_X_FORWARDED_HOST', 'False')
 ALLOWED_HOSTS = os.getenv('ALLOWED_HOSTS', 'localhost').split(',')
+SECURE_PROXY_SSL_HEADER = ('X-Forwarded-Proto', 'https')
+PROD_DB = os.getenv('PROD_DB')
+PROD_FS = os.getenv('PROD_FS', 'False')
+PROD_EMAIL = os.getenv('PROD_EMAIL', 'False')
+if PROD_MODE and PROD_MODE.lower() == 'true':
```
Comment (NickKania) [inline birdspotter/birdspotter/settings.py:48]: Shouldnt this be or?

**Review by NickKania — CHANGES_REQUESTED** [review]
Documentation should be updated to reflect the changes and there appears to be a small logic issue

File: birdspotter/birdspotter/settings.py (hunk 1 above)
Comment (devinchristianson) [reply birdspotter/birdspotter/settings.py:48]: It's ensuring that it's not null/None before calling `.lower()` - if it's false, it'll short-circuit and skip the .lower()

**Review by NickKania — APPROVED** [review]

**Pull request merged**